client/src/components/Preview/Preview.js

```diff
@@ -108,6 +134,12 @@ Preview.propTypes = {
   itemId: React.PropTypes.number,
   onBack: React.PropTypes.func,
   moreActions: React.PropTypes.arrayOf(React.PropTypes.element),
+  moreActionsPopoverId: React.PropTypes.string,
+};
+
+Preview.defaults = {
+  className: 'campaign-admin__campaign-preview', // Legacy defaults
+  moreActionsPopoverId: 'campaign-preview-popover', // Legacy defaults
```

Comment:
Could these defaults be shifted over to `campaign-admin` entirely? Doesn't look like anything is using the Preview yet and would mean decoupling is complete

Reply:
done.